extensions/prost/private/protoc_wrapper.rs

```diff
@@ -457,6 +466,9 @@ struct Args {
     /// Whether to generate tonic code.
     is_tonic: bool,
 
+    // Whether to put a no_std tag into the generated code.
+    is_nostd: bool,
```

Comment:
```suggestion
    is_no_std: bool,
```

Reply:
Woops :D I updated the PR including docs